e2e-cypress/files/tests/installation/installation.spec.cy.ts

```diff
@@ -1,6 +1,9 @@
 describe('installation e2e tests', () => {
```

Comment:
Maybe the whole content of this file should be similar to `integration.spec.cy.ts`:
```
/* tslint:disable:no-unused-expression */
describe('installation e2e tests', () => {
  // see acceptance.spec.cy.ts for examples
});
```
At least old code could be removed and parts of the title should be changed.